Misc/NEWS.d/next/Security/2017-08-06-14-43-45.bpo-28414.mzZ6vD.rst

```diff
@@ -0,0 +1,2 @@
+The ssl module now correctly supports validating hostnames that contain non-
+ASCII characters.
```

Comment:
Just that IDN is pretty jargony and this is the end-user-oriented news text. I made it "hostnames that contain non-ASCII characters (IDNs)".